LetsDoIt.CustomValueTypes/test/LetsDoIt.CustomValueTypes.Tests/EmailTests.cs

```diff
@@ -0,0 +1,30 @@
+using System;
+using Xunit;
+
+namespace LetsDoIt.CustomValueTypes.Tests
+{
+    using Email;
+
+    public class EmailTests
+    {
+
+        [Theory]
+        [InlineData(null)]
+        [InlineData("")]
+        public void Email_WhenIsNullOrWhiteSpace_ShouldThrowArgumentException(string email)
+        {
+           void Test() => new Email(email); 
```

Comment:
We can improve this test with fluent assertion

Reply:
Done